renpy/python.py

```diff
@@ -1202,7 +1202,10 @@ def py_eval_bytecode(bytecode, globals=None, locals=None): # @ReservedAssignment
 
 
 def py_eval(code, globals=None, locals=None): # @ReservedAssignment
-    if isinstance(code, basestring):
+    if isinstance(code, bytes):
+        code = code.decode("utf-8", "python_strict")
```

Comment:
I don't think there's much of a case for sending bytes here is there? Implicitly decoding any that do (most likely accidentally) seems like it would only end up hindering debugging.